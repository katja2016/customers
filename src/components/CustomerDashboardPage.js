import React from 'react';
import CustomerList from './CustomerList';
import CustomerListFilters from './CustomerListFilters';
import CustomersSummary from './CustomersSummary';

const CustomerDashboardPage = () => (
    <div>
        <CustomersSummary/>
        <CustomerListFilters />
        <CustomerList />
    </div>
);

export default CustomerDashboardPage;
