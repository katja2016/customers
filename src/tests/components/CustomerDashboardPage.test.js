import React from 'react';
import { shallow } from 'enzyme';
import CustomerDashboardPage from '../../components/CustomerDashboardPage';

test('should render CustomerDashboardPage correctly', () => {
    const wrapper = shallow(<CustomerDashboardPage/>);
    expect(wrapper).toMatchSnapshot();
});