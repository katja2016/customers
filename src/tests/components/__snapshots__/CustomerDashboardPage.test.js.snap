// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CustomerDashboardPage correctly 1`] = `
<div>
  <Connect(CustomersSummary) />
  <Connect(CustomerListFilters) />
  <Connect(CustomerList) />
</div>
`;
